src/diff_edit.rs

```diff
@@ -340,37 +411,103 @@ pub fn edit_diff(
 #[derive(Clone, Debug, serde::Deserialize)]
 #[serde(rename_all = "kebab-case")]
 struct MergeTool {
-    /// Program to execute.
+    /// Program to execute. Must be defined; defaults to the tool name
+    /// if not specified in the config.
+    #[serde(default)]
```

Comment:
Nit: better to extract this (and corresponding `get_tool_config()` chagges) to a separate commit.

Reply:
I did a version of this. Let me know if that isn't what you had in mind.